wazo_confd/plugins/registrar/service.py

```diff
@@ -29,7 +34,13 @@ def edit(self, registrar, updated_fields=None):
         # Update all affected devices
         for line in lines:
             if line.device_id not in devices_updated:
-                self.device_updater.update_for_line(line)
+                try:
+                    self.device_updater.update_for_line(line)
+                except NotFoundError:
+                    logger.error(
+                        f'Could not update device "{line.device_id}": device not found'
```

Comment:
f-strings! Nice! :smiley_cat:

Reply:
Actually, you made me realize that we should not use them for logging, because it forces eager evaluation, instead of lazy evaluation with
`logger.error('Could not update device "%s", line.device_id)`